apps/docs/src/content/examples/renderers/Flat text with CSS2D.mdx

```diff
@@ -0,0 +1,78 @@
+In this tutorial, we'll walk you through the process of running an additional Three.js renderer in parallel with Threlte's `<Canvas>`, while still leveraging Threlte's built-in elements. Specifically, you'll learn how to run CSS2DRenderer to add flat labels to objects in a three-dimensional scene.
+
+This example can be easily adapted to use CSS3DRenderer instead, if you want the elements to live "inside" the scene, rather than flat across the surface.
+
+<Example path="renderers/CSS2DRenderer" />
+
+## How does it work?
+
+In this scene, we run two renderers - the default one provided by threlte, and also a new CSS2DRenderer which we'll initialize. Threlte's renderer runs on a canvas element as normal, while our new renderer runs in a `<div>` with absolute positioning on top of it.
```

Comment:
Capitalize Threlte.

Maybe reword:

In this scene, we run two renderers: the default one provided by Threlte, and a new CSS2DRenderer which we will be initializing manually.